### DISCUSSION TITLE
When can the open world be seen?

### DISCUSSION THREAD

**Discussion by 2655251820** [UPBGE/upbge#1391] [Ideas/Features for UPBGE 0.3+]
(⊙﹏⊙) I'm very sorry to ask this question, because I really haven't found a solution. Upbge doesn't seem to create tools for the open world? I only found videos from a long time ago, which is meaningless.
Do you have any plans in this regard? Or plan

**Comment by martinsh** [comment]
Hi, I must ask what do you consider an 'open world tool' ?
Open world is a broad term. If you mean something like terrain tools then, yeah. There is almost no ready to use systems like you see in UE or Unity. Even then best systems are third-party assets. Here is where an UPBGE addon could be useful.

**Comment by lordloki** [comment]
A good tool is Geometry nodes where you can scatter static objects in a random way. Another interesting elements have to be build by the developer using current intregated tools as streaming/loading parts of the world, LOD system, etc.

**Comment by youle31** [comment]
As it's a question/Feature request, I move this to discussions

EDIT: The button disappeared

EDIT2: github team is working to restore this option
